app/bundles/EmailBundle/Entity/StatRepository.php

```diff
@@ -658,19 +658,20 @@ public function checkContactsSentEmail($contacts, $emailId)
     }
 
     /**
-     * @param $emailId
+     * @param array<int> $contacts
+     * @param array<int> $emailIds
      *
      * @return array Formatted as [contactId => sentCount]
      */
-    public function getSentCountForContacts(array $contacts, $emailId)
+    public function getSentCountForContacts(array $contacts, array $emailIds)
```

Comment:
You cannot change public method. Any part of code or plugin code could use it and break it then.
Create new method for it to avoid backward compatibility issue.

Reply:
this function is only used once in excactly this context - that's why i mentioned it in the upgrade.md?